.travis.yml

```diff
@@ -1,26 +1,33 @@
+language: python
+
+python:
+  - "2.7"
+
 addons:
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
-
-language: python
+      - python-lxml  # because pip installation is slow
+      - python-simplejson
 
-python:
-  - "2.7"
+env:
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  - secure: QfWoUk6XnvWbjAylKCyZiQDEhIJIJz5AwniN244iabCos6Whm+FmJiCyPEgmLYBuwC23Fd7/ouOSnd3osok0TWve8byIGS8Lvla5ZKNEc4E520tfZyy5xZCZdQ8bA1Gnco9GGRWhIXarb/aWtEgun8HkEibLjIFF7laWtPQ8Suc=
+  matrix:
+  - LINT_CHECK="1"
+  - TRANSIFEX="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
+  - TESTS="1" ODOO_REPO="OCA/OCB"
 
 virtualenv:
   system_site_packages: true
 
-env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-
 install:
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  - git clone https://github.com/OCA/account-financial-reporting ${HOME}/account-financial-reporting -b ${VERSION}
-  - git clone https://github.com/OCA/bank-payment ${HOME}/bank-payment -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/account-financial-reporting ${HOME}/account-financial-reporting -b ${VERSION}
+  - git clone --depth=1 https://github.com/OCA/bank-payment ${HOME}/bank-payment -b ${VERSION}
```

Comment:
Thanks @max3903. 

What about the python dependencies? Is the pip install still required when external_dependencies are up-to-date?